main.js

```diff
@@ -0,0 +1,83 @@
+// this is for menu implementation
+const menuIcon = document.querySelector('.hamb-icon');
+const menu = document.querySelector('.menu-links');
+const exitIcon = document.querySelector('.hamb-close-btn');
+
+menuIcon.addEventListener('click', () => {
+  menu.style.display = 'flex';
+});
+
+exitIcon.addEventListener('click', () => {
+  menu.style.display = 'none';
+});
+
+// the code below for generate speakers dynamically
+const speakerData = [{
+  img: 'speaker1.jpeg',
+  name: 'ALEXANDER J. DAVIS-GRIMES',
+  about: 'PRODUCT DESIGNER, LINKEDIN',
+  talk: ' CREATE THE CAREER YOU’RE LOOKING FOR A design professional with the ability to communicate and empathize with users.',
+},
+
+{
+  img: 'speaker2.jpg',
+  name: 'ANDREW KUMAR',
+  about: 'DIRECTOR, PLATFORM STRATEGY, CONTENTFUL',
+  talk: '  THE CONSEQUENCES OF UX THEATRE Andrew is a global leader of high-performing and diverse digital teams',
+},
+
+{
+  img: 'speaker3.jpg',
+  name: 'ANDY WELFLE',
+  about: 'UX CONTENT STRATEGY MANAGER, ADOBE',
+  talk: 'MAIN BEDROOMS AND SECONDARY BRANCHES: IT’S TIME TO SCRUB THE LANGUAGE OF OPPRESSION OUT OF OUR PRODUCTS.',
+},
+
+{
+  img: 'speaker4.jpg',
```

Comment:
- [ ] Great job implementing this speaker section in such a smart way. This image is currently broke in your project, you should consider fixing it, this is what it looks like right now:

![image](https://user-images.githubusercontent.com/66967274/142275175-1d56484e-5d27-4bfc-9f9c-ea28e32a217b.png)